### PR TITLE
Tony yang

### PR DESCRIPTION
**Pull request by n0tybynature**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

**Design decision to adjust the Support page and move all items to flush left. Previously the need assistance and contact us was centered.** 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "date-fns": "^2.16.1",
     "dotenv": "^8.2.0",
     "feather-icons-react": "^0.4.3",
+    "knex": "^0.95.6",
```
Comment (salvare736) [inline package.json:31]: Correct if I'm wrong, but I believe there shouldn't be any reason to import 'knex' into the frontend repo, no? Was this an intentional addition? Just double-checking!

**Review by salvare736 — CHANGES_REQUESTED** [review]
Changes look solid, but I was just inquiring about the addition of 'knex' to the frontend dependencies.

File: package.json (hunk 1 above)
Comment (n0tybynature) [reply package.json:31]: Definitely no need for knex. good catch!

**Review by elCarlosSantiago — APPROVED** [review]
Solid changes, make sure not to add . so package-lock file doesnt get uploaded as well.

**Review by salvare736 — APPROVED** [review]
If the 'knex' dependency was removed, then we're okay to merge.

**Pull request merged**